src/find/azext_find/custom.py

```diff
@@ -46,8 +48,27 @@ def process_query(question):
                 print('\033[1m' + current_title + '\033[0m')
                 print(current_snippet)
 
-                if i + 1 < num_results_to_show:
-                    print("\n")
+                print("")
+            feedback = prompt("[\033[1mEnter to close.\033[0m Press + or - to give feedback]:")
+            if feedback in ['+', '-']:
+                print('Wow, you are a true hero!')
+                print("""\
+        O_
+       \033[1m<T>\033[0m`-.
+        \033[1m|\033[0m`-‘
+        \033[1mI\033[0m
+                        """)
```

Comment:
Can I see an example of what this looks like?

Reply:
Is this for colorization?

Reply:
![screen shot 2018-11-01 at 12 18 21 pm](https://user-images.githubusercontent.com/457173/47874153-4972da80-ddd0-11e8-9e4c-07732c4269c3.png)

Reply:
Yes, this should be tested on bash, cmd.exe and Powershell.

Reply:
The new commit works in cmd as expected.

Reply:
I did test on cmd and make it work there, but when I copied the code over, there was a typo, tested again, it should work now :)